services/ui-src/src/components/pages/GetStarted/ReportGetStartedPage.tsx

```diff
@@ -60,6 +65,30 @@ export const ReportGetStartedPage = ({ reportType }: Props) => {
               <Text sx={sx.additionalInfo}>{section2.additionalInfo}</Text>
             </Box>
           </Flex>
+          {section2.table ? (
+            <Flex sx={sx.sectionContent}>
+              <Box>
+                <Heading sx={sx.smallSectionHeading} size={"sm"}>
+                  {section2.tableHeading}
+                </Heading>
+                <Table border={true} content={section2.table}></Table>
+              </Box>
+            </Flex>
+          ) : (
+            <></>
+          )}
```

Comment:
Do you still need this? I think if you make line 68 `section2.table &&` instead of `?` you can get rid of this extra bit